packages/scripts/utils/config.js

```diff
@@ -17,7 +17,24 @@ const hasJestConfig = () =>
 	hasProjectFile( 'jest.config.json' ) ||
 	hasPackageProp( 'jest' );
 
+const hasWebpackConfig = () => hasCliArg( '--config' ) ||
+	hasProjectFile( 'webpack.config.js' ) ||
+	hasProjectFile( 'webpack.config.babel.js' );
+
+const getWebpackArgs = ( additionalArgs = [] ) => {
+	const webpackArgs = [];
+	if ( hasWebpackConfig() ) {
+		webpackArgs.push( ...getCliArgs() );
+	} else {
+		webpackArgs.push( ...[ '--config', fromConfigRoot( 'webpack.config.js' ) ] );
+	}
+	webpackArgs.push( ...additionalArgs );
+	return webpackArgs;
+};
+
 module.exports = {
+	getWebpackArgs,
 	hasBabelConfig,
 	hasJestConfig,
+	hasWebpackConfig,
```

Comment:
It looks like `hasWebpackConfig` is never used outside of this file.

Reply:
0ebb4ed